src/document-click.js

```diff
@@ -262,17 +274,25 @@ export function onDocumentElementClick_(
   // The choice of `location.replace` vs `history.replaceState` is important.
   // Due to bugs, not every browser triggers `:target` pseudo-class when
   // `replaceState` is called. See http://www.zachleat.com/web/moving-target/
-  // for more details.
-  win.location.replace(`#${hash}`);
+  // for more details. Do this only if fragment has changed.
+  if (tgtLoc.hash != curLoc.hash) {
+    win.location.replace(`#${hash}`);
```

Comment:
I don't think it's a good idea to have both document tag and share-tracking hash together in the url hash. Maybe we could only have document tag in this case. So I think this is good for now.